tools/javadoc.bzl

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2016 The Android Open Source Project
```

Comment:
please move this file into tools/rules - where we keep all the bazel rule files. It is not ideal in the new version of bazel but eventually when we move to next version of bazel we can move this around.